docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,164 @@
+#######################
+edX Accessibility Guide
+#######################
+
+EdX measures and evaluates accessibility using the World Wide Web Consortium's `Web Content Accessibility Guidelines (WCAG) 2.0 <http://www.w3.org/TR/WCAG/>`_ (Dec. 11, 2008).  All features merged into edX are expected to `conform <http://www.w3.org/TR/WCAG20/#conformance>`_ to `Level AA <http://www.w3.org/TR/WCAG20/#cc1>`_ of this specification.  This guide is intended to extend the guidance available in WCAG 2.0, with a focus on features frequently found in Open edX platform as well as Learning and Content Management Systems in general.
```

Comment:
I can't tell for sure, but are there 2 spaces after periods? Typically we go with 1 space after periods.